vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java

```diff
@@ -137,7 +137,7 @@ public void savePathFromUrl(Set<VariableScope> scopes, String variable)
      * </ul>
      * @param variable A name under which the value should be saved
      */
-    @When("I get the number of open windows and set it to the $scopes variable '$variable'")
+    @When("I get number of open windows and set it to $scopes variable `$variable`")
```

Comment:
we need to do global renaming: `window` -> `tab` (and this should be done within one commit/PR)